packages/x-date-pickers/src/DateCalendar/PickersSlideTransition.tsx

```diff
@@ -10,8 +10,15 @@ import {
   pickersSlideTransitionClasses,
   PickersSlideTransitionClasses,
 } from './pickersSlideTransitionClasses';
+import { PickerOwnerState } from '../models/pickers';
+import { usePickerPrivateContext } from '../internals/hooks/usePickerPrivateContext';
 
 export type SlideDirection = 'right' | 'left';
+
+interface PickerSlideTransitionOwnerState extends PickerOwnerState {
```

Comment:
It is not used in any `styleOverrides`, only in `useUtilityClasses` but I'm also unifying the parameters of this hook